integritybackend/fs_watcher.py

```diff
@@ -109,7 +109,6 @@ def _schedule(
         if handler_class is None:
             raise ValueError(f"Could not find handler class for action {action}")
 
-        path = self.asset_helper.path_for(collection_id, action)
```

Comment:
path is passed in as a parameter
`self, collection_id: str, action: str, patterns: list[str], path: str`

This overwrite "input" as "action-archive"

Looks like action-archive puts passes its own correct path